Libraries/PeriphDrivers/Source/UART/uart_revb.c

```diff
@@ -1021,35 +1131,38 @@ int MXC_UART_RevB_TransactionDMA(mxc_uart_revb_req_t *req)
     MXC_UART_DisableInt((mxc_uart_regs_t *)(req->uart), 0xFFFFFFFF);
     MXC_UART_ClearFlags((mxc_uart_regs_t *)(req->uart), 0xFFFFFFFF);
 
-    MXC_UART_ClearTXFIFO((mxc_uart_regs_t *)(req->uart));
-    MXC_UART_ClearRXFIFO((mxc_uart_regs_t *)(req->uart));
+    // Clearing the RX FIFOs here makes RX-only or TX-only
```

Comment:
Did you mean to leave this commented out? TBH we should probably talk about whether or not we want to clear FIFOs in the other transaction functions as well. That may be outside the scope of this PR though.